assets/src/scripts/monitoring-location/components/daily-value-hydrograph/selectors/legend-data.spec.js

```diff
@@ -1,6 +1,6 @@
-import {lineMarker, rectangleMarker, textOnlyMarker} from 'd3render//markers';
```

Comment:
How did this still work??

Reply:
You mean with the double slashes? I think it just ignored it. Some things seem to be picky about that put clearly the rollup code is not .